src/lib/core/env.js

```diff
@@ -1,8 +1,7 @@
 /* global __dirname module process require */
 
 const {execSync} = require('child_process');
-const {delimiter} = require('path');
-const {joinPath} = require('../utils/utils.js');
+const {delimiter, join} = require('path');
```

Comment:
Not a big fan of extracting these off of path. `delimiter` and `join` are concepts used in a lot more ways than just path, so it'll get a bit confusing down below.

Reply:
Given the usage context (args and neighboring variables/literals always have something do to w/ paths), I thought it wasn't too confusing, but willing to reconsider.